test/Maybe/Maybe.js

```diff
@@ -65,6 +67,7 @@ var RightArb = function(arb) {
 var Compose = function(F, G) {
   var _Compose = function _Compose(x) {
     return {
+      constructor: _Compose,
```

Comment:
👍 It'd be nice to only have to support that method on the constructor, but such is life.